packages/checkout/widgets-lib/src/lib/provider/injected/register.ts

```diff
@@ -0,0 +1,25 @@
+import { EIP1193Provider } from '../types/eip1193';
+
+export interface Register {}
+
+export type DefaultRegister = {
+  /** The EIP-1193 Provider. */
+  provider: EIP1193Provider;
+  /** Reverse Domain Name Notation (rDNS) of the Wallet Provider. */
+  rdns: 'com.coinbase' | 'com.enkrypt' | 'io.metamask' | 'io.zerion';
```

Comment:
out of interest, where do we get these from?

Reply:
I just saw them on the EIP-6963 window post message, but they are also similar to the domain listed on walletconnect explorer https://explorer.walletconnect.com/coinbase-wallet